README.md

```diff
@@ -0,0 +1,63 @@
+## Cloudify ARIA Extensions
+
+This repository enables ARIA to utilize some of Cloudify's abilities, such as orchestrating using AWS and Openstack.
```

Comment:
It's not quite about "utilizing Cloudify's abilities", but rather provide support for using Cloudify plugins on top of ARIA.